situp-plugins/apmtracesource/src/main/java/com/amazon/situp/plugins/source/oteltracesource/OTelTraceSource.java

```diff
@@ -54,30 +57,36 @@ public void start(Buffer<Record<ExportTraceServiceRequest>> buffer) {
         }
         try {
             server.start().get();
-            LOG.info("Started otel_trace_source...");
         } catch (ExecutionException ex) {
-            LOG.error("otel_trace_source failed to start due to:", ex.getCause());
+            if (ex.getCause() != null && ex.getCause() instanceof RuntimeException) {
+                throw (RuntimeException) ex.getCause();
+            } else {
+                throw new RuntimeException(ex);
+            }
         } catch (InterruptedException ex) {
-            LOG.error("otel_trace_source start got interrupted:", ex);
             Thread.currentThread().interrupt();
             throw new RuntimeException(ex);
         }
+        LOG.info("Started otel_trace_source...");
     }
 
     @Override
     public void stop() {
         if (server != null) {
             try {
                 server.stop().get();
-                LOG.info("Stopped otel_trace_source.");
             } catch (ExecutionException ex) {
-                LOG.error("otel_trace_source failed to stop due to:", ex.getCause());
+                if (ex.getCause() != null && ex.getCause() instanceof RuntimeException) {
```

Comment:
The reason for doing these checks is to present the correct exception as possible, avoid wrappers.